### PR TITLE
Add GitHub workflow for php projects with tests

### PR DESCRIPTION
**Pull request by vukanac**
Run tests in CI on PR

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate --strict
```
Comment (vukanac) [inline .github/workflows/php.yml:24]:
```suggestion
      run: php --version && composer --version && composer validate --strict
```

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Validate composer.json and composer.lock
+      run: php --version && composer --version && composer validate --strict
```
Comment (vukanac) [inline .github/workflows/php.yml]:
```suggestion
      run: php --version \
              && composer --version && composer validate --strict \
              && php composer.phar --version && php composer.phar validate --strict
```

File: .github/workflows/php.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      id: composer-cache
+      uses: actions/cache@v3
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress
```
Comment (vukanac) [inline .github/workflows/php.yml]:
```suggestion
      run: php composer.phar install --no-progress
```

File: .github/workflows/php.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```
Comment (vukanac) [inline .github/workflows/php.yml:15]:
```suggestion
    runs-on: php-latest
```

**Comment by MichelHartmann** [comment]
I'm closing this, as it's obsolete due to https://github.com/flyeralarm/php-code-validator/pull/28.

**Pull request closed without merging**